R/expand.R

```diff
@@ -150,8 +150,8 @@ cross_df <- function(x, y) {
   y_idx <- rep(seq_len(nrow(y)), nrow(x))
   dplyr::bind_cols(x[x_idx, , drop = FALSE], y[y_idx, , drop = FALSE])
 }
-drop_empty <- function(x) {
-  empty <- map_lgl(x, function(x) length(x) == 0)
+drop_empty <- function(x, factor=TRUE) {
```

Comment:
Can you please follow http://style.tidyverse.org/syntax.html#spacing and put space around `=` and` before `,`?

Reply:
Also I don't _think_ this actually needs to be an option, but it does need a comment mentioning that we need to keep it consistent with `ulevels()`

Reply:
Yes, of course.  The option there is due to the nesting discussion below (nesting has different behavior).